spec/models/invoice_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Invoice, type: :model do
+  describe 'DB table' do
+      it { is_expected.to have_db_column :date }
+      it { is_expected.to have_db_column :due_date }
+  end
+
+  describe 'Associations' do
+    it { is_expected.to belong_to :user }
+  end
+
```

Comment:
you need to test that the factory is valid

Reply:
done